lib/faker.js

```diff
@@ -176,8 +172,9 @@ function createFakerSchema(schema) {
       return s
     }
     case 'RemoteDetectionAlert':
+      // @ts-ignore
       s.properties.geometry = {
-        $ref: Geometry.$id,
+        //$ref: Geometry.$id,
```

Comment:
I reverted this change from https://github.com/digidem/mapeo-mock-data/commit/03eb40c380fdf1af9bbd77f1d29859e8b89266d8 since keeping `$ref` returns an invalid object (it doesn't respect the geometry schema...)